src/routes/userRoutes.ts

```diff
@@ -0,0 +1,19 @@
+import { Router, Request, Response } from 'express';
+
+const router = Router();
+
+// @ts-ignore
```

Comment:
I faced an issue with an unused parameter `_req: Request` that couldn't find the solution, so in order to avoid TypeScript fails I use // @ts-ignore.